### PR TITLE
Fix parsing of template literals with an escape

### PR DESCRIPTION
**Pull request by nifgraup**
Example: \`\»\`

### CODE REVIEW

**Comment by nifgraup** [comment]
What file would be a good candidate to add a test for a parser bug?

**Comment by fabiosantoscode** [comment]
test/mocha/template-string.js would be nice

File: test/fuzz.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ var minify = require("..").minify;
 var known_terser_errors = new RegExp([
   "Cannot negate a statement",
   "Cannot read property 'references' of undefined",
-  "Cannot read property 'type' of null",
```
[the remark this answers is not in the document]
Comment (nifgraup) [reply test/fuzz.js]: This pull request fixes that error:

```
$ echo "\`\»\`" | bin/uglifyjs
ERROR: Cannot read property 'type' of null
    at type (/terser/lib/parse.js:535:50)
    at cont (/terser/lib/parse.js:715:24)
    at Object.read_template_characters [as input] (/terser/lib/parse.js:758:31)
    at input (/terser/lib/parse.js:901:56)
    at peek (/terser/lib/parse.js:906:24)
    at next (/terser/lib/parse.js:895:15)
    at Object.parse [as minify] (/terser/lib/minify.js:156:42)
    at run (/terser/bin/uglifyjs:215:27)
    at Socket.<anonymous> (/terser/bin/uglifyjs:177:9)
    at emitNone (events.js:111:20)
```

The line was removed so the fuzzer can catch that error if it ever arises again.

[the remark this answers is not in the document]
Comment (fabiosantoscode) [reply test/fuzz.js]: Got it

**Comment by fabiosantoscode** [comment]
Cool stuff, thank you very much!

**Pull request merged**